README.md

```diff
@@ -20,17 +31,18 @@ The `dbt-database-adapter-scaffold` package is a user-friendly interactive way t
 - Fork and clone this repo
 - Once cloned run cookiecutter path_to_this_file/..
 
-#### Generally
-- read the [docs](https://docs.getdbt.com/docs/contributing/building-a-new-adapter) around building a new adapter and follow helpful tips within generated project to build out your new adapter.
-
-## Testing
-Testing for adapter connection capablilities
-- Please use the `dbt-adapter-tests` these are best for troubleshootting connection and common functionality.
+ ### Testing
+A suggested process for testing while building out your new adapter might be something like.
+- Firstly focus on getting the connection methods and credentials setup correctly and test basic connection with `dbt debug`
+- Creating a small project to slowly run and test against locally as you build (will really help once you get to building out unit/integration test implementations) ex. [jaffle_shop](https://github.com/dbt-labs/jaffle_shop) is one we have various instructions and tutorials for (Tutorial)(https://docs.getdbt.com/tutorial/setting-up)
+- Work on building up higher-level macros/methods using your small project from step before to test basic adapter functionality by running things like `dbt run` against building views, tables, `dbt test` till you have basic functionality.
+- Start incorporating new more `pytest` testing to check against your basic adapter functionality. At this point your adapter will be feature-complete at baseline level.
+- Share your adapter to get feedback.
+- Continue developing additional features standard to other dbt adapters and adding in more feature based `unit` and `functional` tests or building out adapter specific test functionality/tests.
```

Comment:
I think this section is mostly covered in https://docs.getdbt.com/docs/contributing/building-a-new-adapter. I'm sorry if I missed other conversation on the PR, but I'd be happy to just link to the docs on the website here.

Reply:
@kwigley I think that's totally fair! I'm the one who asked for this, since there was a stub here previously.

I'm most interested in making clear that the order of operations is _not_:
1. scaffold a new adapter
2. run the testing suite
3. stuff fails, confusion

This more detailed, narrative guidance could definitely want to live in our docs instead, namely https://docs.getdbt.com/docs/contributing/building-a-new-adapter.

@dataders Could I ask you to make the final call on What Goes Where? And help see this PR + https://github.com/dbt-labs/docs.getdbt.com/pull/1362 through?